src/main.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, version):
         self.version = version
 
         GLib.set_application_name("Mousai")
+        GLib.set_prgname('io.github.seadve.Mousai')
```

Comment:
Is this necessary for it to work? As far as I know, this is no longer needed on GTK4 so I removed it before.

Reply:
Blanket is still on GTK3, so it may be that

Reply:
Yes, it is necessary for icon visibility in XFCE's dock. Without it, the StartupWMClass is `python3` and the xfcedock cannot detect it. After this, the StartupWMClass becomes `io.github.seadve.Mousai`

Reply:
Okay then, let's leave it